Sources/ApolloCodegenLib/FileManager+Apollo.swift

```diff
@@ -6,23 +6,10 @@ import ApolloUtils
 
 public typealias FileAttributes = [FileAttributeKey : Any]
 
-/// A protocol to decouple `ApolloExtension` from `FileManager`. Use it to build objects that can support
-/// `ApolloExtension` behavior.
-public protocol FileManagerProvider {
-  func fileExists(atPath path: String, isDirectory: UnsafeMutablePointer<ObjCBool>?) -> Bool
-  func removeItem(atPath path: String) throws
-  @discardableResult func createFile(atPath path: String, contents data: Data?, attributes attr: FileAttributes?) -> Bool
-  func createDirectory(atPath path: String, withIntermediateDirectories createIntermediates: Bool, attributes: FileAttributes?) throws
-}
-
 /// Enables the `.apollo` etension namespace.
 extension FileManager: ApolloCompatible {}
 
-/// `FileManager` conforms to the `FileManagerProvider` protocol. If it's method signatures change both the protocol and
-/// extension will need to be updated.
-extension FileManager: FileManagerProvider {}
-
-extension ApolloExtension where Base: FileManagerProvider {
+extension ApolloExtension where Base: FileManager {
```

Comment:
This should be able to be imported from either `ApolloCore` or ApolloCodegenLib` (I forget which it's declared in) without having to re-declare it.

Reply:
I don't quite understand what you mean. This is the only place where the extension for `FileManager` is declared; it's the same [currently in `main`](https://github.com/apollographql/apollo-ios/blob/main/Sources/ApolloCodegenLib/FileManager%2BApollo.swift#L9). This change is essentially reverting what I'd done with `FileManagerProvider`.

Reply:
Oh derp, I got confused, this IS in `ApolloCodegenLib`. I read good.